### PR TITLE
Add extract_id_dob function to utils

### PR DESCRIPTION
**Pull request by NicolaasDieBaas**
- [x] add function
- [x] add test(s)

```
extract_id_dob: function(id) {
        return moment(id.slice(0,6), 'YYMMDD').format('YYYY-MM-DD');
    },
```

### CODE REVIEW

**Comment by NicolaasDieBaas** [comment]
@imsickofmaps @gsvr please review.  you'll see one commented out test... when '540924'  gets extracted from id number, year of date is set to 2054 instead of 1954.  I assume a correct century fix (moment.parseTwoDigitYear override) needs to be applied whenever we use this function (> '68 currently evaluates to 1900's)

File: test/test_utils.js [hunk 1]
```diff
@@ -320,6 +320,33 @@ describe("Testing utils functions", function() {
         });
     });
 
+    describe("extract_id_dob", function() {
+        it("valid dates extracted", function() {
+            assert.deepEqual(utils.extract_id_dob("8104267805280"),
+                moment("1981-04-26").format("YYYY-MM-DD"));
+            assert.deepEqual(utils.extract_id_dob("8202017805280"),
+                moment("1982-02-01").format("YYYY-MM-DD"));
+            // assert.deepEqual(utils.extract_id_dob("5409237805280"),
```
Comment (imsickofmaps) [inline test/test_utils.js]: You can remove this. If we have 88 year old mothers in the system I think we'll be on TV!

File: lib/utils.js [hunk 2]
```diff
@@ -317,6 +317,17 @@ get_clean_first_word = function(user_message) {
...
 };
 
+// ID HELPERS
+
+/**: extract_id_dob(id)
+Extracts the date of birth from a South African identification number
+
+:param string id: id number
+*/
+extract_id_dob = function(id) {
```
Comment (imsickofmaps) [inline lib/utils.js]: please make this `extract_za_id_dob` - this system is pan-Africa now, we may need extraction functions per country

**Comment by imsickofmaps** [comment]
minor naming then 👍  when travis happy

**Pull request merged**